test/tables.markua

```diff
@@ -0,0 +1,58 @@
+Simple table with caption:
+
+| Right | Left | Center | Default |
+|------:|:-----|:------:|---------|
+|    12 | 12   |   12   | 12      |
+|   123 | 123  |  123   | 123     |
+|     1 | 1    |   1    | 1       |
+
+Demonstration of simple table syntax.
+
+Simple table without caption:
+
+| Right | Left | Center | Default |
+|------:|:-----|:------:|---------|
+|    12 | 12   |   12   | 12      |
+|   123 | 123  |  123   | 123     |
+|     1 | 1    |   1    | 1       |
+
+Simple table indented two spaces:
+
+| Right | Left | Center | Default |
+|------:|:-----|:------:|---------|
+|    12 | 12   |   12   | 12      |
+|   123 | 123  |  123   | 123     |
+|     1 | 1    |   1    | 1       |
+
+Demonstration of simple table syntax.
+
+Multiline table with caption:
+
+| Centered Header | Left Aligned | Right Aligned | Default aligned                                       |
+|:---------------:|:-------------|--------------:|:------------------------------------------------------|
+|      First      | row          |          12.0 | Example of a row that spans multiple lines.           |
+|     Second      | row          |           5.0 | Here’s another one. Note the blank line between rows. |
+
+Here’s the caption. It may span multiple lines.
+
+Multiline table without caption:
+
+| Centered Header | Left Aligned | Right Aligned | Default aligned                                       |
+|:---------------:|:-------------|--------------:|:------------------------------------------------------|
+|      First      | row          |          12.0 | Example of a row that spans multiple lines.           |
+|     Second      | row          |           5.0 | Here’s another one. Note the blank line between rows. |
+
+Table without column headers:
+
+|     |     |     |     |
+|----:|:----|:---:|----:|
+|  12 | 12  | 12  |  12 |
+| 123 | 123 | 123 | 123 |
+|   1 | 1   |  1  |   1 |
+
+Multiline table without column headers:
+
+|        |     |      |                                                       |
+|:------:|:----|-----:|-------------------------------------------------------|
```

Comment:
Does markua support this way of indicating alignment in pipe tables with colons?  Not every implementation does.

Reply:
Yes Markua supports this [feature](https://leanpub.com/markua/read#tables)